lit/docs/operation/tracing.lit

```diff
@@ -8,11 +8,16 @@ Tracing in Concourse enables the delivery of traces related to the internal
 processes that go into running builds, and other internal operations, breaking
 them down by time, and component.
 
-It currently integrates with \link{Jaeger}{https://www.jaegertracing.io/} and
-\link{Google Cloud Trace}{https://cloud.google.com/trace} (Stackdriver),
-although support for other systems is planned to expand as the underlying SDK
-(\link{OpenTelemetry}{https://opentelemetry.io/}) evolves.
+It leverages the (\link{OpenTelemetry}{https://opentelemetry.io/}) SDK to allow support for many
+platforms. Currently tracing can be configured to integrates with:
 
+\list{
+  \link{Jaeger}{https://www.jaegertracing.io/}
+}{
+  \link{Google Cloud Trace}{https://cloud.google.com/trace} (Stackdriver)
+}{
+  \link{OpeneTelemetry Collector Exporter}{https://github.com/open-telemetry/opentelemetry-go/tree/master/exporters/otlp}
```

Comment:
In my mind "opentelemetry collector" refers to a specific program, but your PR allows exporting to anything that speaks OTLP -- I thought it stood for "opentelemetry line protocol", but then i read [the docs](https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md).
```suggestion
  \link{OpenTelemetry Protocol Exporter}{https://github.com/open-telemetry/opentelemetry-go/tree/master/exporters/otlp}
```